wiki/Options/en.md

```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
 | Name | Description |
 | --- | --- |
 | `VSync` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
-| `120fps` | Limits the game client to 120fps. |
-| `240fps` | Limits the game client to 240fps. |
+| `Power Saving` | Limits the game client to twice the refresh rate of your monitor. |
+| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
 | `Unlimited (gameplay)` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
 
 - `VSync` explanation: In layman terms, using VSync will force the game wait for the entire frame to load before displaying it.
   - You may want to use this if you see some "tearing" (when the bottom portion of the game is lagging behind the upper portion of the game).
   - Despite what the point above stated, this may cause some lag or slowdowns because the game client has to wait for each frame to load.
 - `Unlimited (gameplay)` explanation: Unlimited only applies to when you are playing a beatmap.
-  - When you are doing anything but playing a beatmap, `240fps` is used instead.
+  - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.
```

Comment:
2x monitor refresh rate now

Reply:
It seems to be still limiting to the 240fps, is it a 4x refresh rate? (cuttingedge 20191207.1)

Reply:
2x but lowest possible is 240, i guess

Reply:
```suggestion
  - When you are not playing a beatmap, the frame rate is limited to twice the refresh rate of your monitor, or 240fps, whichever is higher.
```